app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            "INSERT INTO actors (first_name, last_name) "
+            "VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        cinema_cursor = self._connection.execute(
+            "SELECT *"
+            "FROM actors"
```

Comment:
There is a missing space between `SELECT *` and `FROM actors`. This will cause a syntax error. Consider changing it to `SELECT * FROM actors`.